src/mgmt/srv_target.c

```diff
@@ -144,6 +146,7 @@ destroy_cb(const char *path, const struct stat *sb, int flag,
 		rc = unlink(path);
 	if (rc)
 		D_ERROR("failed to remove %s\n", path);
+	sched_yield();
```

Comment:
Same here, the destroy_cb() could be called by a request processing ULT on xstream 0, then the whole xstream 0 will give up CPU unexpectedly.

Reply:
ok. Will remove this.